Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Omise\Payment\Model\Ui;
+
+use Magento\Checkout\Model\ConfigProviderInterface;
+use Magento\Payment\Api\PaymentMethodListInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Omise\Payment\Model\Capabilities;
+use Omise\Payment\Model\Config\Installment as OmiseInstallmentConfig;
+
+class CapabilitiesConfigProvider implements ConfigProviderInterface
```

Comment:
So since this class is providing configs specifically for Installment, should we rename this one to just `InstallmentConfigProvider`?

Reply:
not sure... 

Capabilities, we provide here capabilities for all payment methods in the future?